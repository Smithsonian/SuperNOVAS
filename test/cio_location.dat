
2450545.000000 TEST       S2 O0 A1: 2    -0.002338

2450545.000000 TEST       S2 O1 A1: 2    -0.002358

2450545.000000 TEST       S2 O2 A1: 2    -0.002338

2450545.000000 SUN        S0 O0 A1: 2    -0.002338

2450545.000000 SUN        S0 O1 A1: 2    -0.002338

2450545.000000 SUN        S0 O2 A1: 2    -0.002338

2451545.000000 TEST       S2 O0 A1: 2     0.000000

2451545.000000 TEST       S2 O1 A1: 2     0.000000

2451545.000000 TEST       S2 O2 A1: 2     0.000000

2451545.000000 SUN        S0 O0 A1: 2     0.000000

2451545.000000 SUN        S0 O1 A1: 2     0.000000

2451545.000000 SUN        S0 O2 A1: 2     0.000000

2452545.000000 TEST       S2 O0 A1: 2     0.002339

2452545.000000 TEST       S2 O1 A1: 2     0.002052

2452545.000000 TEST       S2 O2 A1: 2     0.002339

2452545.000000 SUN        S0 O0 A1: 2     0.002339

2452545.000000 SUN        S0 O1 A1: 2     0.002339

2452545.000000 SUN        S0 O2 A1: 2     0.002339
