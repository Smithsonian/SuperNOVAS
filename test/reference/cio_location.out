  -10000.000 A0: 2 -0.023103017 2 -0.023103017 
       0.000 A0: 2 -0.000236403 2 -0.000236403 
   10000.000 A0: 2  0.023566385 2  0.023566385 
   10000.000 A0: 2  0.023566385 2  0.023566385 
   10000.010 A0: 2  0.023566416 2  0.023566416 
  -10000.000 A1: 2 -0.023103017 2 -0.023103017 
       0.000 A1: 2 -0.000236402 2 -0.000236402 
   10000.000 A1: 2  0.023566385 2  0.023566385 
   10000.000 A1: 2  0.023566385 2  0.023566385 
   10000.010 A1: 2  0.023566416 2  0.023566416 
